addons/binding/pom.xml

```diff
@@ -34,6 +34,7 @@
     <module>org.openhab.binding.bluetooth.bluez</module>
     <module>org.openhab.binding.bluetooth.blukii</module>
     <module>org.openhab.binding.bluetooth.test</module>
+    <module>org.openhab.binding.bluetooth.ruuvitag</module>
```

Comment:
Better move the module above the `test` as the `r` is before the `t`.

Reply:
Fixed in 6aea086